contentcuration/contentcuration/frontend/administration/pages/Users/EmailUsersDialog.vue

```diff
@@ -1,164 +1,151 @@
-                    lazy
+              <template #item="{ item }">
+                <div class="chip-container">
+                  <KTooltip
+                    :refs="getRefs()"
+                    :reference="`tooltip-${item.id}`"
+                    placement="bottom"
                   >
-                    <template #activator="{ on }">
-                      <VChip
-                        small
-                        :close="recipients.length > 1"
-                        data-test="remove"
-                        v-on="on"
-                        @input="remove(item.id)"
-                      >
-                        <div style="max-width: 72px">
-                          <div class="text-truncate">
-                            {{ item.name }}
-                          </div>
-                        </div>
-                      </VChip>
-                    </template>
                     <span>{{ item.name }} &lt;{{ item.email }}&gt;</span>
-                  </VTooltip>
-                </template>
-              </ExpandableList>
-              <VChip
-                v-else-if="usersFilterFetchQueryParams"
-                small
-              >
-                {{ searchString }}
-              </VChip>
-            </VFlex>
-          </VLayout>
-          <VTextField
-            v-model="subject"
-            class="mt-4"
-            box
-            label="Subject line"
-            required
-            :rules="requiredRules"
-          />
-          <div class="caption grey--text">Add placeholder to message</div>
-          <div class="mb-1">
-            <VBtn
-              v-for="placeholder in placeholders"
-              :key="`placeholder-${placeholder.label}`"
-              small
-              round
-              depressed
-              color="grey lighten-4"
-              style="text-transform: none"
-              @click="addPlaceholder(placeholder.placeholder)"
-            >
-              {{ placeholder.label }}
-            </VBtn>
+                  </KTooltip>
+                  <StudioChip
+                    :ref="`tooltip-${item.id}`"
+                    :text="item.name"
+                    :close="recipients.length > 1"
+                    data-test="remove"
+                    @close="remove(item.id)"
+                  >
+                    <div class="text-truncate">
+                      {{ item.name }}
+                    </div>
+                  </StudioChip>
+                </div>
+              </template>
+            </ExpandableList>
+            <StudioChip v-else-if="usersFilterFetchQueryParams">
+              {{ searchString }}
+            </StudioChip>
           </div>
-          <VTextarea
+        </div>
+        <KTextbox
+          v-model="subject"
+          class="mt-4"
+          label="Subject line"
+          :required="true"
+          :invalid="errors.subject"
+          :showInvalidText="showInvalidText && errors.subject"
```

Comment:
[`showInvalidText` configures](https://design-system.learningequality.org/ktextbox#validation) if
> shows the validation text even if the user has not focused or changed the input

So it's not clear to me why there's the need to set `showInvalidText` dynamically. Can't it simply be set to `true` at all times if that's what you're trying to achieve?